### PR TITLE
Add history box button to header

### LINKED ISSUE #979: History Box: Add another way to open the history box
The history box currently can be opened only through a button in the itemnav. The history box also is not closed when the circuit is locked (which is the correct, intended behavior).
Those two facts cause a weird incongruity in that it is not possible to open the history box while the circuit is locked, even though it is allowed.
There should be another button somewhere else (probably either the header or the sidenav) to open the history box to correct this.

### PR DESCRIPTION
**Pull request by EricW123**
Fixes #979.

Added a new history button at top left corner, next to locker button.

### CODE REVIEW

File: src/site/shared/containers/Header/Left/Locker.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {useSharedDispatch, useSharedSelector} from "shared/utils/hooks/useShared";
+import {ToggleCircuitLocked} from "shared/state/CircuitInfo";
+
+import "./index.scss";
+
+export const LockToggleButton = () => {
```
Comment (TGCrystal) [inline src/site/shared/containers/Header/Left/Locker.tsx:6]: nit: 2 newlines between last import and start of code

File: src/site/shared/containers/Header/Left/SideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {useSharedDispatch} from "shared/utils/hooks/useShared";
+import {ToggleSideNav} from "shared/state/SideNav";
+
+import "./index.scss";
+
+export const SideBarToggleButton = () => {
```
Comment (TGCrystal) [inline src/site/shared/containers/Header/Left/SideBar.tsx:6]: nit: 2 newlines between last import and start of code

File: src/site/shared/containers/Header/Left/Locker.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    const {isLocked} = useSharedSelector(
+        state => ({ ...state.circuit })
+    );
+    const dispatch = useSharedDispatch();
+
+	return (
+		<div>
```
Comment (TGCrystal) [inline src/site/shared/containers/Header/Left/Locker.tsx:13]: nit: Move this `<div>` one indentation to the right

File: src/site/shared/containers/Header/Left/SideBar.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    const dispatch = useSharedDispatch();
+
+	return (
+		<div>
```
Comment (TGCrystal) [inline src/site/shared/containers/Header/Left/SideBar.tsx:10]: nit: one indentation to the left

File: src/site/shared/containers/Header/Left/SideBar.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+            <span title="Side Bar" role="button" tabIndex={0}
+                  onClick={() => dispatch(ToggleSideNav())}>&#9776;</span>
+        </div>
```
Comment (TGCrystal) [inline src/site/shared/containers/Header/Left/SideBar.tsx:13]: nit: one indentation to the right

**Review by TGCrystal — CHANGES_REQUESTED** [review]
You will need to adjust the breakpoint for the dropdown arrow in the top right that appears on mobile. As it is now, the top right dropdown buttons disappear at 700px but the arrow dropdown of dropdowns doesn't appear until 600px.

**Review by LeonMontealegre — APPROVED** [review]

**Pull request merged**